pyproject.toml

```diff
@@ -38,6 +38,7 @@ dependencies = [
     "scipy >= 1.2",  # <https://github.com/scipy/scipy/pull/9176>
     "pyyaml",  # For YAML headers of plaintext datafiles.
     "meshio",  # For reading VTK and other meshes.
+    "cmcrameri",  # Matplotlib colormaps using the palettes designed by Fabio Crameri.
```

Comment:
I would suggest [palettable](https://jiffyclub.github.io/palettable/) instead. It includes more maps.

Reply:
Interesting, I didn't see that one. Feels a bit more clunky to use though, no `batlow`, only `Batlow_20` etc. Not sure I need 700 cmaps, I just want the good ones :) Plus I cannot find a way to forward declare cmaps using string names so I'd have to import the `palettable.scientific.sequential` everywhere.